Reports/Requirement Specification/Requirement Specification.tex

```diff
@@ -281,7 +293,7 @@ \subsection{Non-functional Requirements}
 {Operational Requirement}
 {The software shall be documented clearly and the user manual should detail the functional aspects and maintenance of the system}
 {High - For our client to be satisfied with our work, and for it to have potential in research, we need to ensure that it can be operated without our assistance}
-{The mark awarded to us by our client for the quality of our communication in the ``Product Handover'' section}
```

Comment:
for future reference, latex won't properly format quote marks or inverted commas around text. to do it you have to use backticks <code>`</code> for opening and inverted commas <code>'</code> for closing